docs/api/public_php_api_managing_repository.md

```diff
@@ -140,7 +140,7 @@ foreach ($workflowMetadata->markings as $marking) {
 
 !!! tip
 
-    `marking`, a term from [Symfony Workflow,](https://symfony.com/doc/5.0/components/workflow.html)
+    `marking`, a term from [Symfony Workflow,]([[= symfony_doc =]]/components/workflow.html)
```

Comment:
This is actually an ugly hack done on purpose, otherwise the punctiation mark lands after the external link icon 
![image](https://user-images.githubusercontent.com/13621049/126304844-bbbbe037-cea7-4f39-85dd-83723a39562e.png)